cpr/CMakeLists.txt

```diff
@@ -53,3 +54,12 @@ else()
         ${CPR_INCLUDE_DIRS}
         ${CURL_INCLUDE_DIRS})
 endif()
+
+
+# install headers
+install(DIRECTORY "${CPR_INCLUDE_DIRS}/cpr/" DESTINATION include/cpr FILES_MATCHING PATTERN "*.h")
```

Comment:
Isn't this achieved with [this](https://github.com/whoshuu/cpr/blob/00d87d8e447802f88218bf826f4359c9ffeeb80c/CMakeLists.txt#L83)?